config/functions

```diff
@@ -278,11 +278,27 @@ setup_toolchain() {
     TARGET_LDFLAGS+=" $FLAGS_OPTIM_LTO_OFF"
   fi
 
+  # Linker flags
+  if [ "${DEFAULT_LINKER}" = "" ]; then
+    TARGET_LINKER_FLAGS="-fuse-ld=bfd"
+  else
+    TARGET_LINKER_FLAGS="-fuse-ld=${DEFAULT_LINKER}"
+  fi
+
+  # bfd flag
+  if flag_enabled "bfd" "no"; then
+    TARGET_LINKER_FLAGS=" ${LDFLAGS_OPTIM_BFD}"
+  fi
   # gold flag
-  if flag_enabled "gold" "$GOLD_SUPPORT" "only-disable"; then
-    TARGET_LDFLAGS+=" $LDFLAGS_OPTIM_GOLD"
+  if flag_enabled "gold" "no"; then
+    TARGET_LINKER_FLAGS=" ${LDFLAGS_OPTIM_GOLD}"
     have_gold="yes"
   fi
+  # mold flag
+  if flag_enabled "mold" "no"; then
+    TARGET_LINKER_FLAGS=" ${LDFLAGS_OPTIM_MOLD}"
+  fi
+  TARGET_LDFLAGS+=" ${TARGET_LINKER_FLAGS}"
```

Comment:
This should be `if mold..elif gold..elif bfd` (or case)? Only one should be set per package.

Reply:
updated